pkgs/applications/office/appflowy/default.nix

```diff
@@ -13,11 +13,11 @@
 
 stdenv.mkDerivation rec {
   pname = "appflowy";
-  version = "0.3.8";
+  version = "0.4.1";
 
   src = fetchzip {
-    url = "https://github.com/AppFlowy-IO/appflowy/releases/download/${version}/AppFlowy-${version}-linux-x86_64.tar.gz";
-    hash = "sha256-3ICeKSqzx1zp/KpaAFl9qLSaugWm4HZrKjrDCWz9ok4=";
+    url = "https://github.com/AppFlowy-IO/appflowy/releases/download/${version}/AppFlowy-x86_64-unknown-linux-gnu-x86_64.tar.gz";
```

Comment:
The link returns HTTP 404.

https://cache.nixos.org/log/3bmpmjpssd5414vnksy72bkc7m02ahaz-source.drv
https://github.com/AppFlowy-IO/AppFlowy/releases/tag/0.4.1

Reply:
Thanks to point that out, I had a bad feeling when updating the links. I should have waited a bit more, since they're kinda used to upload, then rename their assets. 
Never mind, I fixed that in #279173